src/main/java/com/helloworld/springboot/HelloWorldController.java

```diff
@@ -9,11 +9,11 @@
 @Controller
 public class HelloWorldController {
 
-    @RequestMapping("helloworld")
+@RequestMapping("helloworld")
```

Comment:
```suggestion
@GetMapping("/helloworld")
```
がいいですね！
HTTPメソッドのGETとは何かについても少し調べてみてください！！
https://developer.mozilla.org/ja/docs/Web/HTTP/Methods